docs/toc.yml

```diff
@@ -982,6 +982,11 @@ items:
     - name: Word
       href: quickstarts/word-quickstart-vs.md
       displayName: Word
+  - name: Tutorials
+    items:
+    - name: PowerPoint
```

Comment:
Since PowerPoint is isolated here and users might not be able to distinguish between quick starts and tutorials, could we give the PPT tutorial a scenario-focused name? Maybe "Work with a presentation"? (It's hard because the tutorial shows off a sort of random subset of APIs - which may be a larger issue.)